home.mdx

```diff
@@ -3,51 +3,35 @@ title: Meilisearch Documentation
 description: Here you'll find everything you need to learn and create with our lightning fast search engine.
 ---
 
-# Welcome to the Meilisearch documentation
+# Documentation
 
-Here you'll find everything you need to learn and create with our lightning fast search engine.
+Explore our guides and references to learn how to use Meilisearch in your projects.
 
-<Capsule className="my-[72px]">
-  <h3 className="text-3xl">Looking for SDK documentation?</h3>
-  <p>Check out this <ButtonLink href="/docs/learn/what_is_meilisearch/sdks.html" currentColor="true" >list of official Meilisearch</ButtonLink> libraries or use the Integrations dropdown menu at the top of the screen for quick access.</p>
-</Capsule>
-
-<h2 className="text-5xl">Meilisearch news</h2>
-
-Announcing Meilisearch Cloud: [register now](https://cloud.meilisearch.com/register) or [log in here](https://cloud.meilisearch.com/login) if you already have an account.
+<Capsule>
+  ## Quick start
 
-<Divider />
-
-We just raised a 15M Series A! 🥳 Check out the [TechCrunch exclusive](https://docs.meilisearch.com/#:~:text=TechCrunch%20exclusive,opens%20new%20window) or learn more [on our blog](https://blog.meilisearch.com/meilisearch-series-a/).
+  New here? Check out the [Quick start guide](/learn/getting_started/quick_start.mdx) and learn how to launch a basic Meilisearch instance.
+</Capsule>
 
-<Divider />
+<h2 className="text-5xl">Explore The docs</h2>
 
-<h2 className="text-5xl">Explore our documentation</h2>
 <Featured items={[
   {
     icon: 'cube',
-    content: 'Check what we will be working on in the next few weeks!'
+    content: 'Consult the Meilisearch [API reference](/reference/api).'
   },
   {
-    content: 'Check what we will be working on in the next few weeks!',
+    content: 'Looking for SDK documentation? Check out this list of [official Meilisearch libraries](/learn/what_is_meilisearch/sdks.mdx).',
     icon: 'compass'
   },
   {
     content:
-      'Check out this list of official Meilisearch libraries or use the Integrations dropdown menu at the top of the screen for quick access.',
-    icon: 'user'
+      'Announcing [Meilisearch Cloud](https://cloud.meilisearch.com/register): the best way to add Meilisearch to your project',
```

Comment:
```suggestion
      '[Meilisearch Cloud](https://cloud.meilisearch.com/register): the easiest way to add Meilisearch to your project',
```

Reply:
I thought we were trying to avoid words like "easy" and "simple"?

Reply:
We do avoid those words in a documentation context, but I don't think we ever decided to avoid them in a marketing context.

Reply:
Either way, this suggestion is pretty subjective so I'm fine leaving it as is.